File: curso-python/ex086.py
matriz = [
    #coluna 0 # coluna 1 #coluna 2
    [0,0,0], # linha 0 
    [0,0,0], # linha 1
    [0,0,0]  # linha 2
]

for linha in range(0,3):
    for coluna in range(0,3):
        matriz[linha][coluna] = int(input(f"Digite um valor para [{linha},{coluna}]: "))

for linha in matriz:
    for valor in linha:
        print(f"[ {valor} ] ", end='')
    print()

File: javascript-exercicios/ex086.js
const prompt = require("prompt-sync")();

let matriz = [
    [0,0,0],
    [0,0,0],
    [0,0,0]
];

for (let linha = 0; linha < 3; linha++){
    for(let coluna = 0; coluna < 3; coluna++){
        matriz[linha][coluna] = Number(prompt(`Digite um valor para [${linha}, ${coluna}]: `));
    }
}

for (const linha of matriz){
    let lF = ""
    for (const valor of linha){
        lF += `[ ${valor} ]`
    }
    console.log(lF.trim())
}